Add projects by AngryCarrot

AngryCarrot789 opened the pull request:
added som things

AngryCarrot789 committed:
Create touchscreen.lua

AngryCarrot789 committed:
Create README.md

tomodachi94 committed:
Delete README.md

Tomodachi94 commented:
A few things are missing, most notably:
- Missing LICENSE. Are you OK with using the MIT License?
- Not compliant with standards I am about to push. Please move your file into a directory structure: `src/<Author>/<Project>/main.lua` Highly recommend using `main.lua` to be consistent.

AngryCarrot789 committed:
Rename touchscreen.lua to Carrot/TouchScreen/touchscreen.lua

AngryCarrot789 committed:
Rename touchscreen.lua to touchscreen_base.lua

AngryCarrot789 commented:
mit licence is ok

tomodachi94 committed:
Rename touchscreen_base.lua to main.lua

tomodachi94 committed:
Create LICENSE

Approved by owner, see https://github.com/Tomodachi94/CondemnedCC/pull/1#issuecomment-766463299

Tomodachi94 commented:
Looks good, merging!
